.github/workflows/ci.yml

```diff
@@ -646,7 +646,7 @@ jobs:
         arch: [armeabi-v7a, arm64-v8a, x86, x86_64]
 
     runs-on: ubuntu-22.04
-    container: ghcr.io/f3d-app/f3d-ci
+    container: ghcr.io/f3d-app/f3d-android-${{ matrix.arch }}:${{needs.default_versions.outputs.vtk_commit_sha}}
```

Comment:
what happens if the container does not exists ?

Reply:
It will fail until the Docker images are available, so it needs a PR in `f3d-docker-images`.  
I know it's not ideal, I'm planning to improve it, but at least it ensures VTK sync.

Reply:
afaiu f3d-docker-images builds the image name based on f3d master, so that cant work ?

Reply:
nop: https://github.com/f3d-app/f3d-docker-images/blob/237b9cfc157f99550d056fd95706666afe4db400/.github/workflows/build_docker_image.yml#L12

Reply:
Yes that's what I had in mind.